.travis.yml

```diff
@@ -19,3 +21,9 @@ before_script:
 
 script:
   - npm test
+
+matrix:
+  allow_failures:
```

Comment:
True. Usually I check Travis' page anyway rather than wait for notifications, so I figured I didn't want things to fail (yet) if it was just 0.12 or io.js, especially on this first run. I'm probably gonna keep the `allow_failures` in for a short while and then take them out at a later date.